### PR TITLE
Update actions, apply coding style

### PR DESCRIPTION
**Pull request by Gerschtli**
Found new command `nix fmt` and https://github.com/nerdypepper/statix, quite nice to catch some inconsistencies and stuff. Also updated the github workflows. What do you think?

### CODE REVIEW

**Review by t184256 — COMMENTED** [review]
Formatting: opinionated, but OK

Actions: r+

flake.lock: why is nixpkgs pinned to a path?

**Comment by Gerschtli** [comment]
> Actions: r+

what does r+ mean?

> flake.lock: why is nixpkgs pinned to a path?

my bad, didn't want to commit that one

**Comment by t184256** [comment]
> what does r+ mean?

positive review

**Comment by Gerschtli** [comment]
Pipeline is having trouble with `flake.lock`. Some general thought to that: How about adding a `flake.lock` because when we start building the bootstrap zip with flake setup we will either way add a lock file and when using nix-on-droid as input, one should always override nixpkgs and home-manager inputs, so should not break anything and would improve portability because nix tools always expect a flake.lock. What do you think?

**Comment by t184256** [comment]
> one should always override nixpkgs and home-manager inputs

I dislike the consequences of forgetting to do that, but I do agree that avoiding adding a flake.lock is not a proper solution.

I guess we can add it.

**Comment by Gerschtli** [comment]
@t184256 PTAL

File: flake.nix [hunk 1]
```diff
@@ -40,7 +41,13 @@
...
-      defaultApp.aarch64-linux = app;
-    };
+
+      apps.aarch64-linux = {
+        default = app;
+        nix-on-droid = app;
+      };
+    }
+    // flake-utils.lib.eachDefaultSystem (system: {
+      formatter = nixpkgs.legacyPackages.${system}.nixpkgs-fmt;
```
Comment (t184256) [inline flake.nix:51]: Nit: why do we export it?

**Review by t184256 — APPROVED** [review]
Approved in general.

File: flake.nix (hunk 1 above)
Comment (Gerschtli) [reply flake.nix:51]: nix flake show won't show it, but it is necessary for nix fmt to work.

Comment (t184256) [reply flake.nix:51]: Huh, OK then.

**Comment by t184256** [comment]
Don't forget to autosquash before merging.

**Pull request merged**